### DISCUSSION TITLE
[Feature] Be able to merge recognized faces

### DISCUSSION THREAD

**Discussion by hakanu** [immich-app/immich#3128] [Feature Request]
### The feature

First of all thanks for this great app, I was using photoprism and it had tons of problems.

One thing happening to me is that facial recognition is getting different people for the same person. Is it possible to merge them? I know this is a bit of a small thing with complicated implementation. However, would be really nice touch.

Thanks.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #2472
